docs/whats-new/2019-11.md

```diff
@@ -0,0 +1,171 @@
+---
+title: ".NET Docs What's New (November 2019)"
+description: "What's new in .NET docs in November 2019."
+ms.date: 2019/12/13
+---
+
+# .NET Docs What's New (November 2019)
+
+Welcome to what's new in .NET docs for November 2019. Listed below are some of the recent docs we've published.
+
+## .NET Core
```

Comment:
I see .NET Core and .NET Framework, but not .NET, i.e., .NET Standard. Was the .NET guide overlooked, or were there no significant changes in it?

Reply:
This time, I didn't find any significant changes.